templates/event_checkin.html.twig

```diff
@@ -26,8 +26,15 @@
 {% block javascripts %}
     <script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
     <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.0/jquery-ui.min.js"></script>
-
+    <script src="/js/main.js" type="text/javascript" charset="utf-8"></script>
+    <script type="text/javascript" charset="utf-8">
+        $(function() {
+            Raffler.init();
```

Comment:
Couldn't we pass the list of winners to that function? Then we wouldn't expose the `winners` property.